File: leetcode/easy/js/832.js
const solution = (A) => {
  let result = []
  for (let i = 0; i < A.length; i++) {
    let tempArr = []
    for (let j = A[i].length - 1; j >= 0; j--) {
      let currentItem = A[i][j]
      if (currentItem === 1) {
        currentItem = 0
      } else {
        currentItem = 1
      }
      tempArr.push(currentItem)
    }
    result.push(tempArr)
  }
  return result
}

const input = [[1, 1, 0], [1, 0, 1], [0, 0, 0]]

console.log(solution(input))

File: leetcode/by_level/easy/py/832.py
def solution(A):
  result = []
  for i in range(len(A)):
    tempArr = []
    for j in range(len(A[i]) - 1, -1, -1):
      currentItem = A[i][j]
      if currentItem == 1:
        currentItem = 0
      else:
        currentItem = 1
      tempArr.append(currentItem)
    result.append(tempArr)
  return result

input = [[1, 1, 0], [1, 0, 1], [0, 0, 0]]
print(solution(input))